knowledge_graph.py

```diff
@@ -0,0 +1,110 @@
+"""Knowledge Graph"""
+from typing import Dict, List, Any
+from transformers import (
+    Text2TextGenerationPipeline,
+    AutoModelForSeq2SeqLM,
+    AutoTokenizer)
+from transformers.pipelines.text2text_generation import ReturnType
+from pydantic import Field
+
+class KnowledgeGraph():
+  """Knowledge Graph class"""
+
+  max_new_tokens: int = Field(
+    default=200, description="Maximum number of new tokens to generate."
+  )
+  predictions : List[Dict[str, Any]] = Field(default=[])
+
+  def __init__(
+    self,
+    *args,
+    model_name:str='Babelscape/rebel-large',
+    **kwargs):
+
+    self.model = EntityRelationshipExtractionPipeline(model=model_name, *args, **kwargs)
+
+  def inference(self, sentence: str) -> Dict[str, Any]:
+    """inference"""
+    self.predictions = self.model(sentence)
+
+    return self.predictions
+
+class EntityRelationshipExtractionPipeline(Text2TextGenerationPipeline):
```

Comment:
**Suggestion:** Add type hints to the `EntityRelationshipExtractionPipeline` class.
```suggestion
+class EntityRelationshipExtractionPipeline(Text2TextGenerationPipeline):
+  def __init__(self, model: str, *args, **kwargs) -> None:

```